### ISSUE TITLE
`package-member-without-void` rule incorrectly flags correct member shape after compile-time merge introduction

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#1273]
The rule at `src/main/resources/org/eolang/lints/design/package-member-without-void.xsl:15` reports every named top-level formation in a `+package` file that declares no void attribute:

```xml
<xsl:for-each select="/object/o[eo:abstract(.) and @name and not(o[eo:void(.)])]">
```

and tells the programmer the object "does not declare any void attributes, so it can never be reached in the implicit (receiver) form, only through the package namespace". That was true while `PhDefault.absent` was the only way a member could get a receiver: at runtime it looked the member up on the classpath and wrote the receiver into the member's first void, so a member with no void had nowhere for one to go.

objectionary/eo#6616 replaces that lookup with a compile-time merge. A new `4-merge` stage splices every member of a package into the object the package names, and the member then reads its receiver through `^`:

```
[] > naturals
  range > @
    ...
  ^ > r
```

Declaring no void is the correct shape for such a member, not a defect, and the rule reports it anyway. Worse, it cannot tell the two cases apart: it runs at `3-lint`, one stage before the merge, and whether a package is merged is decided by the `mergedPackages` parameter of the merge mojo, which no XMIR the linter reads carries. The question the rule asks stopped being answerable where it is asked.

objectionary/eo#6770 added the first three `+unlint package-member-without-void` metas, to `range/naturals.eo`, `chunk/to-output.eo` and `output/dead.eo`. Seven packages are left in objectionary/eo#6657 and each one adds more, on the way to roughly forty metas whose only purpose is to say that a rule no longer applies.

Delete the stylesheet and the six packs under `src/test/resources/org/eolang/lints/packs/single/package-member-without-void/`, along with the rule's entry wherever the lint set enumerates it. The behaviour it guarded is now the compiler's to enforce, and it has the merge stage to enforce it with.

**Comment by yegor256** [comment]
One more data point from objectionary/eo#6856, which strips every `+unlint`
meta out of the `.eo` sources: 11 of the 99 metas it removes are
`package-member-without-void`, in `chunk/to-output`, `directory/deleted`,
`directory/made`, `file/deleted`, `file/touched`, `input/as-bytes`,
`input/dead`, `input/length`, `output/dead`, `range/naturals` and
`tuple/is-empty`. Every one of those members already reads its receiver through
`^`, so there is nothing to fix in any of them.

With the metas gone, `eo-runtime` has to name the rule in `<skipSourceLints>`
in its pom instead, which switches it off for all 171 sources rather than the
11 that trip it. That is a worse place for the suppression to live than a meta
was, and it stays that way until the stylesheet is deleted.

**Comment by yegor256** [comment]
@maxonfjvipon let's close this one?

**Comment by 0crat** [comment]
@maxonfjvipon Thanks for closing this issue! You've earned +5 points for this: +8 as a basis; -3 for 7 days of delay. Your running score is [+986](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
